src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -32,8 +32,18 @@ class ObjectField extends React.Component {
     super();
     this.isRequired = this.isRequired.bind(this);
     this.orderAndFilterProperties = _.flow(
-      properties =>
-        orderProperties(properties, _.get('ui:order', this.props.uiSchema)),
+      properties => {
+        const order = _.get('ui:order', this.props.uiSchema);
```

Comment:
I don't think we need the `_.get()` here since it's only one level deep and we don't have a default.

Reply:
Yeah, I think `_.get` was originally used to either use a default value or for if `uiSchema` didn't exist. Seeing as we're not using a default value (just `undefined`) and we can use optional chaining if `uiSchema` doesn't exist, I can remove this. :+1: